### PR TITLE
[JBDS-3263] Removing JSF visual editor

### PR DESCRIPTION
**Pull request by dgolovin**
Part of work related to https://issues.jboss.org/browse/JBDS-3263

### CODE REVIEW

File: jsf/features/org.jboss.tools.jsf.feature/feature.xml [hunk 1]
```diff
@@ -20,8 +20,7 @@
 
    <requires>
       <import feature="org.jboss.tools.common.feature" version="3.6.0" match="greaterOrEqual"/>
-      <import feature="org.jboss.tools.jst.feature" version="3.6.0" match="greaterOrEqual"/>
-      <import feature="org.jboss.tools.vpe.feature" version="3.3.0" match="greaterOrEqual"/>
+      <import feature="org.jboss.tools.jst.feature" version="3.6.0" match="greaterOrEqual"/>OC
```
Comment (scabanovich) [inline jsf/features/org.jboss.tools.jsf.feature/feature.xml]: Are letters 'OC' needed at the end of the line?

File: jsf/plugins/org.jboss.tools.jsf.vpe.ajax4jsf/src/org/jboss/tools/jsf/vpe/ajax4jsf/ajax4jsf-palette.xml [hunk 2]
```diff
@@ -1,210 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (scabanovich) [inline jsf/plugins/org.jboss.tools.jsf.vpe.ajax4jsf/src/org/jboss/tools/jsf/vpe/ajax4jsf/ajax4jsf-palette.xml]: This is a part of JSF palette. May we still need it for Source editor in pages?

**Comment by maxandersen** [comment]
is this issue still relevant  ? seems outdate now ?

**Comment by alexeykazakov** [comment]
If I understand it correctly and we don't remove the JSF visual editor entirely (only for Mac users) then this PR is outdated. Rejecting.

**Pull request closed without merging**